### PR TITLE
WIP: Example: List custom IOA Rule Groups

### PR DESCRIPTION
**Pull request by isimluk**

### CODE REVIEW

**Comment by isimluk** [comment]
How to test:
 - check out this branch
 - `go run ./examples/falcon_custom_ioas`

/cc @falcon-pioupiou

File: specs/transformation.jq [hunk 1]
```diff
@@ -38,4 +38,6 @@
   | .paths."/cloud-connect-cspm-azure/entities/download-certificate/v1".get.parameters[1].default = "false"
 
   # Needed by rusty-falcon (stricter typing)
-  | .definitions."deviceapi.DeviceDetailsResponseSwagger".properties.errors."x-nullable" = true
\ No newline at end of file
+  | .definitions."deviceapi.DeviceDetailsResponseSwagger".properties.errors."x-nullable" = true
+
+  | .paths."/ioarules/queries/rule-groups-full/v1".get.responses."200" = .paths."/ioarules/entities/rule-groups/v1".get.responses."200"
```
Comment (isimluk) [inline specs/transformation.jq:43]: Hey @ffalor,

The PR is includes the changes to transformation, but it is missing the code change in the generated code.

File: examples/falcon_custom_ioas/main.go [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+				empty = false
+			}
+
+			fmt.Println(json)
+		}
+	}
+	fmt.Println("]")
+}
+
+func getIoaRuleGroupDetails(client *client.CrowdStrikeAPISpecification, ruleGroupIds []string) []*models.APIRuleGroupV1 {
```
Comment (isimluk) [inline examples/falcon_custom_ioas/main.go:72]: Would you mind removing this method. It is no longer called.

Comment (isimluk) [reply examples/falcon_custom_ioas/main.go:72]: Oh, I am resolving this comment, I haven't realised that it is still being used in this version of the file.

File: specs/transformation.jq (hunk 1 above)
Comment (ffalor) [reply specs/transformation.jq:43]: sorry about that - its added

**Pull request merged**